### ISSUE TITLE
Getting warnings when adding a layer 

### ISSUE THREAD

**Issue by hanbyul-here** [tangrams/tangram#452]
- In [this routing example](http://mapzen.github.io/lrm-mapzen/tangram-route.html), I get a lot of warnings after mapzen.js bumped up Tangram version to v0.11.0 
warning message is : 
```
mapzen.com/tangram/0.11/tangram.debug.js:40779 Tangram v0.11.0 [warn]: tile load error(s) for routes/15/5240/12668/15: undefinedlog @ mapzen.com/tangram/0.11/tangram.debug.js:40779(anonymous function) @ mapzen.com/tangram/0.11/tangram.debug.js:41992WorkerBrokerMainThreadHandler @ mapzen.com/tangram/0.11/tangram.debug.js:42053

```
- The example above converts routing result to geojson, and dynamically add route data as a layer. [code is here](https://github.com/mapzen/lrm-mapzen/blob/gh-pages/js/tangram-route.js#L86-L123)

**Comment by bcamper** [comment]
Should be resolved by https://github.com/tangrams/tangram/commit/1e59bdc52855ce7cd04259e69e511b0c4b746a7e, for next release.

**Comment by bcamper** [comment]
Released in v0.11.1.

**Comment by hanbyul-here** [comment]
Hi, I just checked the [example](http://mapzen.github.io/lrm-mapzen/tangram-route.html) with v0.11.1 in Chrome and it gets `Uncaught TypeError: Cannot read property 'postMessage' of undefined`

**Comment by bcamper** [comment]
Looking into it - it's a regression related to 7a4c47921c419f9ee7dc8b16c0b21b84dcc99341, which was a change made to reduce loading non-tiled data sources multiple times (previously they were loaded once per worker, which is wasteful).

But in both of these cases (the original issue, and the regression), there is something about this demo that causes a race condition or unexpected initialization state in Tangram... so we will need to look closer to better understand why and prevent further issues.

**Comment by bcamper** [comment]
@hanbyul-here I believe this should be resolved with https://github.com/tangrams/tangram/commit/e2428b1e24afba13b29bb6b04827553b4e0eec5d, can you please verify locally with a custom mapzen.js build?

**Comment by hanbyul-here** [comment]
@bcamper just checked with local build. The problem seems to be solved. (I see route line and console is clean ✨ ) Thanks!

**Comment by bcamper** [comment]
Should be re-resolved in newly released 0.11.3 :)
